Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -32,6 +32,10 @@ def loadPatientByUID(patientUID):
       for patientUID in patientUIDs:
         loadedNodeIDs.extend(DICOMUtils.loadPatientByUID(patientUID))
 
+    Note that since there is no way to have a globally unique patient ID,
```

Comment:
Hmm, I think the rest of the comment actually already says exactly what you describe.  I said "db.patients() returns values unique for the current database" and you said "this patient UID is the internal unique identifier used in the database".

In any case I added this comment since most people have trouble with it, and we really should use patientUID since it's not a valid dicom concept.  So we should no doubt explain it more clearly.  I'll rework the comment.